static/js/info_overlay.js

```diff
@@ -79,4 +82,17 @@ exports.maybe_show_keyboard_shortcuts = function () {
     exports.show("keyboard-shortcuts");
 };
 
+exports.render_markdown = () => {
+    let unrendered_text;
+    let obj;
+    $.each($(".apply_markdown"), (id, element) => {
```

Comment:
Let's scope this selector to just inside the overlay.

Reply:
Also, could we just rename in a prep commit the `preserve_spaces` class to `markdown_source`, and then avoid adding a duplicate `apply_markdown` class that will appear in all the same places?